repo/packages/H/hdfs/0/config.json

```diff
@@ -11,7 +11,8 @@
           "type": "string"
         },
         "default": [
-          "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/artifacts/dcos/0.1.1/hdfs-mesos-0.1.1.tgz"
+          "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/artifacts/dcos/0.1.1/hdfs-mesos-0.1.1.tgz",
```

Comment:
Great question @gabrielhartmann. You can ping me, and I will walk you through the upgrade process once the comments are addressed. You can merge this PR in Universe after the other PR is merged and we release a new version of HDFS. First step will be getting a team city account from Ben W.